docs/SlavaUkraini/examples/accessibility/index.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: tutorial_v2
+title: A guide to basic Leaflet accessibility
+---
+
+## Accessible maps
+
+Web accessibility is the inclusive practice of ensuring no barriers exist
```

Comment:
Add a link here to MDN accesibility pages, maybe?

Reply:
Good suggestion, thanks, done.